bdb/attr.h

```diff
@@ -645,6 +645,9 @@ DEF_ATTR(DEBUG_TIMEPART_SQLITE, dbg_timepart_SQLITE, BOOLEAN, 0, NULL)
 DEF_ATTR(DELAY_FILE_OPEN, delay_file_open, MSECS, 0, NULL)
 DEF_ATTR(DELAY_WRITES_IN_RECORD_C, delay_writes_in_record_c, MSECS, 0, NULL)
 
+DEF_ATTR(DOHSQL_VERBOSE, dohsql_verbose, BOOLEAN, 0, NULL)
+DEF_ATTR(DOHAST_VERBOSE, dohast_verbose, BOOLEAN, 0, NULL)
+DEF_ATTR(DOHSQL_DISABLE, dohsql_disable, BOOLEAN, 0, NULL)
```

Comment:
Can you please add a description for these?

Reply:
hm, there is no way to provide description for bdb/attr.h parameters.  I have switched them to tunables.